### PR TITLE
Write preliminary product description

### LINKED ISSUE #1: Write README Description
Summary of press release / why-this-tool

### PR DESCRIPTION
**Pull request by hannahhenderson**
Resolves #1

### CODE REVIEW

**Comment by hannahhenderson** [comment]
Write preliminary description of project

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # divvy
-Hack Reactor Greenfield Project - Expense Splitting App
+#### Hack Reactor Greenfield Project - Expense Splitting App
+
+## Description
+Divvying up expenses between friends and peers can be a disaster. Remember that time you went on a trip and one person rented the car and another paid for dinners and a third bought the gas and the beer and then you had to even-up while accounting for tips? Or that time rent was due but someone had purchased all of the groceries and utilities for the month? Enter Divvy.
```
Comment (dting) [inline README.md:5]: Is this sentence better off with commas instead of "and"s?

Comment (hannahhenderson) [inline README.md:5]: I'm happy to add commas, I was trying to convey the unpleasant nature of divvying expenses up by using a run-on

Comment (dting) [inline README.md:5]: Cool. That sounds like a good reason to keep it the way it is. LGTM.

**Pull request merged**